core/src/main/java/com/amplitude/core/Configuration.kt

```diff
@@ -30,6 +30,7 @@ open class Configuration @JvmOverloads constructor(
     open var identifyBatchIntervalMillis: Long = IDENTIFY_BATCH_INTERVAL_MILLIS,
     open var identifyInterceptStorageProvider: StorageProvider = InMemoryStorageProvider(),
     open var identityStorageProvider: IdentityStorageProvider = IMIdentityStorageProvider(),
+    var isNetworkConnected: Boolean = true,
```

Comment:
If we put the variable on the configuration object, should we call it like this or just offline? I guess when it is set here it will also be allowed to change. A side thing is if we add it in the core configuration to allow custom set then we might also need to add that in the android configuration.

Reply:
I am open to the property name here. @justin-fiedler wdyt?

> I guess when it is set here it will also be allowed to change. A side thing is if we add it in the core configuration to allow custom set then we might also need to add that in the android configuration.

Nice catch. I move it outside of the constructor in https://github.com/amplitude/Amplitude-Kotlin/pull/171/commits/ea4753e9611e9608ba90548a5dbbb1f04511ddd2